app/components/Table/index.tsx

```diff
@@ -71,6 +72,9 @@ const Table: React.FC<Props> = ({
         // so we can update the route and re-render without suspending
         if (mode === 'replace') router.replace(url, url, { shallow: true });
         else router.push(url, url, { shallow: true });
+
+        // The .split('&')[0] is to remove the filter args since we only want to save sorting
+        cookie.set('analyst.sort', router.asPath?.split('&')[0]);
```

Comment:
Did you test this because I did and sorting is always set with ?orderBy whether it’s set first or after a filter has been applied

Reply:
I meant moreso that this only splits before the first `&`, so if the URL is `?filterArgs=%7B"statusSortFilter"%3A"rec"%7D&orderBy=STATUS_SORT_FILTER_ASC` then the filter args are stored rather than the `orderBy`.

Reply:
For sure, good catch thanks!